### DISCUSSION TITLE
not sure why also firefox, flatpack, evolution... is downloaded @ installation?

### DISCUSSION THREAD

**Discussion by revast** [Security-Onion-Solutions/securityonion#12377] [2.4] [answered]
### Version

2.4.40

### Installation Method

Security Onion ISO image

### Description

installation

### Installation Type

Standalone

### Location

on-prem with Internet access

### Hardware Specs

Meets minimum requirements

### CPU

4

### RAM

32

### Storage for /

300

### Storage for /nsm

600

### Network Traffic Collection

tap

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

No, one or more services are failed (please provide detail below)

### Salt Status

Yes, there are salt failures (please provide detail below)

### Logs

No, there are no additional clues

### Detail

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/8189198/96dedd80-269a-42eb-b31a-0ef1a7ea22fd)
looks like these are downloaded during installation - is this intended?

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by TOoSmOotH** [comment, marked as answer]
Yes. These are updates for security onion desktops if they get added to your grid.
